pkgs/top-level/all-packages.nix

```diff
@@ -32359,6 +32359,8 @@ with pkgs;
     (import ../../nixos/lib/make-options-doc/default.nix)
     ({ inherit pkgs lib; } // attrs);
 
+  options = callPackage ../tools/nix/options {};
```

Comment:
Not sure about his name, wouldn't it be better to call it `nixos-options-flake` or something :thinking: ?

Reply:
changed to nix-option